apps/docs/packages/telegram-apps-mate/hosting.md

```diff
@@ -20,6 +20,30 @@ The hosting service is free, though there are some limitations on data transfer.
 > adjustments. If your app already distinguishes between static and dynamic
 > requests, integration will be much simpler.
 
+
+###	Important Recommendation:
+If your application uses client-side routing (e.g., with SPA frameworks), Mate’s hosting 
+does not support server-side routing methods like `BrowserRouter` from `react-router-dom`. It is recommended 
+to use alternative routing methods, such as `HashRouter`, or configure server-side 
+route handling to ensure proper navigation within your mini application.
+
+```jsx
+import { HashRouter as Router, Route, Switch } from 'react-router-dom';
+
+function App() {
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/" component={Home} />
+        {/* Other routes */}
+      </Switch>
+    </Router>
+  );
+}
+
+export default App;
+```
+
```

Comment:
The problem is we have developers not using React.js as their main library. Could we write something "...so, for instance, if you are using React, use the HashRouter" or kind of it?